### ISSUE TITLE
CAPS/NUM lock & Mod Keys (v1.7.5)

### ISSUE THREAD

**Issue by Fredo699** [symless/synergy#5499]
From what I can tell, just by using this application, when CAPS lock is on, on the server side, and the user is typing text over to the client, synergy wraps the characters typed with CAPS Lock keypresses. So, if you type "A", it will send [Caps Lock Toggle] [A] [Caps Lock Toggle].

The trouble is when you're typing keys that do not have letters attached to them, like the super key or the alt key. When using the GNOME desktop for instance, a single press of the super key should send you into overview mode, but when you have capslock on, GNOME registers your action as CapsLock+Super, not as a sequence of keypresses.

**Comment by XinyuHou** [comment]
@Fredo699 
> GNOME registers your action as CapsLock+Super, not as a sequence of keypresses.
What is the result of this? Does super key not send you into overview mode?

**Comment by Fredo699** [comment]
@XinyuHou 
No, the result is that super key is not sent to the client. I haven't tried it with any DE on the server besides GNOME, though.
